Week-2/HTML-ADVANCED/RESTUARANT-REVIEWS/about.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+</head>
+
+<body>
+<header>
+    <h1>Restaurant Reviews</h1>
+    <nav>
+        <a href="./index.html">Home</a>
+        <a href="./addReview.html">Add Review</a>
+        <a href="#" class="active">About</a>
+    </nav>
+</header>
+<main>
+    <section>
+        <h2>About</h2>
+        <p>Restaurant Reviews is a website that collects user reviews for restaurants.</p>
+    </section>
+    <section>
+        <h2>Disclaimer</h2>
+        <p>All reviews on this website are written by our website users and should not be regarded as an endorsement
+            of
+            any particular restaurant by Restaurant Reviews.</p>
+    </section>
+    <section>
+        <h2>Contact Us</h2>
+        <em>Email: <a href="#">mm@gmail.com</a></em><br>
```

Comment:
To enhance privacy, consider obfuscating email addresses and phone numbers to prevent them from being harvested by bots. You could use JavaScript to dynamically insert this information into the DOM, or alternatively, use a service that provides email obfuscation.

```
<script>
  document.getElementById('email').textContent = 'mm' + '@' + 'gmail.com';
</script>
```
Remember to add an element with` id="email"` where you want the email to display.